linode/manager
# Add copy tooltip to IP addresses on the Linode Networking tab

## Description

We usually have a copy tooltip associated with IP addresses and there's an open issue (https://github.com/linode/manager/issues/7896) asking for it.

## How to test

- Go to the Networking tab on a Linode Detail page
- The copy tooltip should show when the user is hovering over a row
- The copied text should match up with the associated IP address
